gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionReminderScheduler.java

```diff
@@ -142,8 +150,7 @@ public static JobDetail createReminderJobDetail(DagActionStore.DagActionLeaseObj
     dataMap.put(ReminderJob.FLOW_ACTION_EVENT_TIME_KEY, dagActionLeaseObject.getEventTimeMillis());
 
     return JobBuilder.newJob(ReminderJob.class)
-        .withIdentity(createDagActionReminderKey(dagActionLeaseObject.getDagAction()),
-            dagActionLeaseObject.getDagAction().getFlowGroup())
+        .withIdentity(createJobKey(dagActionLeaseObject.getDagAction(), deadlineReminder))
```

Comment:
so the group here is always true or false? true if original deadline event, false for the reminders after trying lease arbitration? can u clarify this at one point above

Reply:
group is set inside the method `createJobKey` and it depends on `deadlineReminder`. does that answer your question?

Reply:
oh yes I see how it's called. Can you check what the group is used for? Will there be any unintended performances consequences of having all the reminders go into a generic "reminder" vs "deadlineTrigger" group? I'm wondering if group is used to filter so filtering by flowGroup would be faster but it may not be noticeable performance loss

Reply:
yes, the use is similar to groups in our flowConfigs, to group similar stuff together and use groups in doing get/delete